.release-please-manifest.json

```diff
@@ -0,0 +1,3 @@
+{
+  ".": "2.5.1"
```

Comment:
The version number should be the last released version in this branch ([doc](https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md#initial-version))